sitesupport/common.js

```diff
@@ -22,6 +23,12 @@ function escapeHtml(text) {
   return text.replace(/[&<>"']/g, function(m) { return map[m]; });
 }
 
+function simpleObjectToRequest(obj) {
```

Comment:
This function may be why you are seeing weird behavior where `null` becomes `"null"` in your query params. I believe JS has some built-in tooling for handling URL encoded parameters in forms that we might consider leveraging.

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
Even still, I believe it's the case that passing in an object with `{ foo: null }` will still result in `foo=null` here. So to support query params we may need to approach it with the magic values.

Reply:
Yeah, for now documented magic values, Tomorrow the world.